src/Documentation/ComponentDocs/ButtonDocs/ButtonDocs.js

```diff
@@ -0,0 +1,191 @@
+      </div>
+      <Gist id="d4a3c1f1f41812aed7a0b5771b2a29da" />
+
+      <h3>Color Schemes</h3>
+      <p>
+        <code>colorScheme</code> prop is used to change the color scheme of the
+        button of any variant. The default value is <code>light green</code>,
+        which doesn't need to be passed. Other available option is{" "}
+        <code>red</code>.
+      </p>
+      <p>
+        <code>Note: </code> Only one color scheme, i.e. red is available as of
+        now. More will be added soon.
+      </p>
+
+      <div className="usage make-flex">
+        <Button variant="secondary" text="Button" />
+        <Button colorScheme="red" variant="secondary" text="Button" />
+      </div>
+      <Gist id="3599ea27eb2868bf89074582c53e24ce" />
+
+      <h3>Disable Buttons</h3>
+      <p>
+        <code>isDisabled</code> prop is used to disable the button. Possible
+        values for this prop is true or false. The default value is{" "}
+        <code>false</code>.
+      </p>
+      <div className="usage make-flex">
+        <Button isDisabled={true} text="Button" />
+        <Button variant="secondary" isDisabled={true} text="Button" />
+      </div>
+      <Gist id="c11aafa00efbf0b44c1a3fe5eaf6ca13" />
+      <h3>Button with Icon</h3>
+      <p>
+        You can add left and right icons to the Button component using the{" "}
+        <code>leftIcon</code> and <code>rightIcon</code> props respectively.
+      </p>
+      <div className="usage make-flex">
+        <Button leftIcon={<HiOutlineMail />} text="Email" />
+        <Button
+          rightIcon={<MdNavigateNext />}
+          text="Next"
+          variant="secondary"
+        />
+      </div>
+      <Gist id="1837a0e69a94e8253b9d079af502b1eb" />
+      <h3>Social Buttons</h3>
+      <p>
+        We've included the colors for common social media platforms, and you can
+        simply use their buttons via the <code>colorScheme</code> prop.
+      </p>
+      <div className="usage make-flex">
+        <Button leftIcon={<FaFacebook />} colorScheme="fb" text="Facebook" />
+        <Button leftIcon={<FaTwitter />} colorScheme="twitter" text="Twitter" />
+      </div>
+      <Gist id="04b7711de27fd76c3cfd1f2f7565c09c" />
+      <p>
+        The Social Media Buttons can have variants and different sizes as well.
+      </p>
+      <div className="usage make-flex">
+        <Button
+          leftIcon={<FaFacebook />}
+          colorScheme="fb"
+          variant="secondary"
+          text="Facebook"
+        />
+        <Button
+          leftIcon={<FaTwitter />}
+          colorScheme="twitter"
+          variant="ghost"
+          text="Twitter"
+        />
+      </div>
+      <Gist id="9f9fe43b38382e1bf6fb1d9b42f1da80" />
+      <h3>Rounded Buttons</h3>
+      <p>
+        <code>borderRadius</code> prop can be used to set the border radius of
+        the buttons. It can take up 4 values - <code>b1, b2, b3, full</code>.
```

Comment:
I have a suggestion here, b1,b2,b3 is not clear, can you please make it a descriptive.

Reply:
Done with this change. Thanks

Reply:
Done making it radius-1,radius-2, radius-3 and circle. Hopefully gives a better idea of what the value of border-radius prop represents.